### PR TITLE
FE-1518 - Preview Component

### PR DESCRIPTION
**Pull request by dawidzarzycki**
# Description
- Update `Preview` component to use `styled-components`

# Screenshots / Gifs
preview

# Testing Instructions
- There are no differences according to theme (classic, small, medium, large are the same)

### CODE REVIEW

File: src/components/preview/preview.spec.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import Preview from './preview';
+import TestRenderer from 'react-test-renderer';
+import 'jest-styled-components';
+import Preview from './preview.component';
 
 const renderShallow = (children, props) => {
```
Comment (grabkowski) [inline src/components/preview/preview.spec.js]: it's not shallow now :)

File: src/components/preview/__snapshots__/preview.spec.js.snap [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      "height": undefined,
+      "width": undefined,
+    }
+  }
+/>
+`;
+
+exports[`Preview when given children will render the children 1`] = `"This is some text"`;
+
+exports[`Preview when given no children and a falsy loading prop will render nothing 1`] = `null`;
```
Comment (grabkowski) [inline src/components/preview/__snapshots__/preview.spec.js.snap]: I think we shouldn't have snapshots like these

File: src/components/preview/__snapshots__/preview.spec.js.snap [hunk 3]
```diff
@@ -0,0 +1,163 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Preview when given a width and height renders with the given width and height 1`] = `
+.c1 {
+  -webkit-animation: blNbyL 2s ease infinite;
+  animation: blNbyL 2s ease infinite;
```
Comment (grabkowski) [inline src/components/preview/__snapshots__/preview.spec.js.snap]: wouldn't that be different on each snapshot generation?

Comment (dawidzarzycki) [reply src/components/preview/__snapshots__/preview.spec.js.snap]: It should works good

File: src/components/preview/__snapshots__/preview.spec.js.snap (hunk 2 above)
Comment (dawidzarzycki) [reply src/components/preview/__snapshots__/preview.spec.js.snap]: Just copied, refactor isn't in the scope

**Pull request merged**